libresapi/src/api/PeersHandler.cpp

```diff
@@ -643,7 +643,7 @@ void PeersHandler::handleWildcard(Request &req, Response &resp)
 			std::string error_string;
 			RsPeerDetails peerDetails;
 
-			RsPgpId own_pgp = mRsPeers->getGPGOwnId();
+			/*RsPgpId own_pgp =*/ mRsPeers->getGPGOwnId();

```

Comment:
I dont see the point in leaving this statement. It's not doing anything, is it? If so, just remove the line.

Reply:
Just in case getGPGOwnId initialize something...
But doesn't seems the case.